src/context/TagsContext.js

```diff
@@ -25,17 +26,31 @@ export function TagsProvider(props) {
   // EVENT HANDLERS
   const loadTags = async () => {
     const tags = await api.getTags();
-    log.debug('load tags from api:', tags.length);
-    setTagList(tags);
+    log.debug('load tags from api:', tags);
+    setTagList(tags.tags);
   };
   /*
    * check for new tags in tagInput and add them to the database
    */
+
   const createTags = async () => {
+    const orgId = getOrganizationUUID();
+    log.debug('orgId:', orgId);
+    const newTagTemplate = {
+      isPublic: orgId ? false : true,
+      owner_id: orgId,
```

Comment:
Just a thought, we will need a way to verify that the user has permission to set the `owner_id` to a particular org on the backend (using our Keycloak integration).

Reply:
Good thought --- I'm making the temporary assumption that keycloak will manage access via permissions and that the user will only have access to approve/reject the organizations that they are allowed to edit.  The tags are currently added to a general table, but they are associated to a particular capture via id on a join table.  So a user can add tags to a capture when they approve it, but they can't edit them directly or without going through a capture that belongs to their approved organization.